slackbot.py

```diff
@@ -0,0 +1,51 @@
+import os
+from slack import RTMClient
+import ssl, certifi
+import requests
+import dotenv
+
+dotenv.load_dotenv('ENV/.env')
+
+# API ENDPOINT
+URL = 'http://localhost:5000/chatbot/'
+
+ssl_context = ssl.create_default_context(cafile=certifi.where())
+
+@RTMClient.run_on(event="message")
+def mapbot(**payload):
+  '''
+  This function is a listener for 
+  message events happening in the 
+  the workspace.
+  '''
+  data = payload['data']
+  web_client = payload['web_client']
+  bot_id = data.get('bot_id', '')
+
+  # Ignore Bot Messages
+  if bot_id == '':
+    channel_id = data['channel']
+    # User message from slack
+    text = data.get('text', '')
+    text = text.split('>')[-1].strip()
+
+    response = ""
+    if 'help' in text.lower():
+      user = data.get('user', '')
+      response = f"Hi <@{user}>! I am Mapbot :)"
+    else:
+      chatbot_response = requests.get(f"{URL}{text}").json()
+      response = chatbot_response[0]['message'][0]
+    
+    # Sends the message to the slack
+    web_client.chat_postMessage(
+        channel=channel_id,
+        text=response
+      )
+
+try:
+  rtm_client = RTMClient(token=os.getenv("SLACK_BOT_TOKEN"), ssl=ssl_context)
```

Comment:
Add a line in `config.py` for fetching `slack_bot_token` and rather use `config.slack_bot_token` here instead of `os.getenv()`